### PR TITLE
Fix clown not being clumsy

### LINKED ISSUE #5201: You can shoot guns as the clown
## Description
Something is wrong with clumsy component, we have failed as a remake

### PR DESCRIPTION
**Pull request by Ephememory**
change addcomponentspecial to serialize componentregistery instead of 1 component
change clumsydamage to just be constant values
Fixes #5201
:cl: Memory
- fix: Made the clown clumsy again.

### CODE REVIEW

File: Content.Server/Weapon/Ranged/ServerRangedWeaponComponent.cs [hunk 1]
```diff
@@ -34,6 +40,7 @@ public sealed class ServerRangedWeaponComponent : SharedRangedWeaponComponent, I
     {
         [Dependency] private readonly IMapManager _mapManager = default!;
         [Dependency] private readonly IGameTiming _gameTiming = default!;
+        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
```
Comment (mirrorcult) [inline Content.Server/Weapon/Ranged/ServerRangedWeaponComponent.cs:43]: remove

**Review by mirrorcult — APPROVED** [review]

File: Content.Server/Jobs/AddComponentSpecial.cs [hunk 2]
```diff
@@ -11,16 +12,27 @@ namespace Content.Server.Jobs
...
+            // now its a registry of components, still throws i bet.
+            var factory = IoCManager.Resolve<IComponentFactory>();
+            var entityManager = IoCManager.Resolve<IEntityManager>();
+            var serializationManager = IoCManager.Resolve<ISerializationManager>();
+
+            foreach (var (name, data) in Components)
+            {
+                var component = (Component) factory.GetComponent(name);
+                component.Owner = mob;
+                _ = serializationManager.Copy(data, component, null);
```
Comment (PaulRitter) [inline Content.Server/Jobs/AddComponentSpecial.cs:31]: use the return value to call addcomponent with

**Review by PaulRitter — CHANGES_REQUESTED** [review]

**Comment by PaulRitter** [comment]
@DrSmugleaf i don't recall: can you copy datadefinitions like that?

File: Content.Server/Jobs/AddComponentSpecial.cs (hunk 2 above)
Comment (Ephememory) [reply Content.Server/Jobs/AddComponentSpecial.cs:31]: return value of which function

Comment (PaulRitter) [reply Content.Server/Jobs/AddComponentSpecial.cs:31]: Copy, the one you are currently writing into _

**Comment by metalgearsloth** [comment]
You coming back to this?

**Comment by Ephememory** [comment]
> You coming back to this?

dont feel like it, no

**Comment by Ephememory** [comment]
i feel like it now

**Pull request merged**